app/models/sample.rb

```diff
@@ -295,6 +296,13 @@ def sample_expt_s3_path
     "s3://#{SAMPLES_BUCKET_NAME}/#{sample_path}/expt"
   end
 
+  def contigs_fasta_s3_path
+    pr = pipeline_runs.first
+    return "#{pr.postprocess_output_s3_path}/#{ASSEMBLY_DIR}/#{CONTIGS_FASTA_BASENAME}" if pr.pipeline_version && pr.pipeline_version.to_f >= 3.1
```

Comment:
in pipeline_run.rb there's ASSEMBLED_CONTIGS_NAME = 'assembly/contigs.fasta' already.
I think you can just do  "#{pr.postprocess_output_s3_path}/#{ASSEMBLED_CONTIGS_NAME}"